x-pack/plugin/esql/qa/testFixtures/src/main/resources/dissect.csv-spec

```diff
@@ -14,6 +14,43 @@ foo bar   | null       | null
 ;
 
 
+shadowing
```

Comment:
Added analogous csv tests to LOOKUP, ENRICH, DISSECT, GROK and EVAL, since they should handle attribute shadowing identically.